src/components/teampage/team-member-styles.js

```diff
@@ -0,0 +1,189 @@
+import styled from "styled-components"
+import Img from "gatsby-image"
+import { devices } from "../../constants/devices"
+
+const fonts = {
+  manrope: `'Manrope Extra Bold', 'Manrope', 'Arial',
+    Arial, sans-serif`,
+  poppinsRegular: `'Poppins Regular', sans-serif`,
+  poppinsBold: `'Poppins Bold', sans-serif`,
+}
+
+const fontSizes = {
+  name: `calc(16px + (38 - 16) * ((100vw - 375px) / (1920 - 375)))`,
+  position: `calc(12px + (20 - 12) * ((100vw - 375px) / (1920 - 375)))`,
+  text: `calc(8px + (20 - 10) * ((100vw - 375px) / (1920 - 375)))`,
+}
+
+const CardPosition = styled.div`
+  min-width: calc(50% - 30px);
+  height: 200px;
+  display: inline-flex;
+  margin-bottom: ${props =>
+    props.isLast && props.index % 2 === 0 ? "0" : "160px"};
+  margin-left: ${props => (props.index % 2 === 0 ? "0" : "60px")};
+
+  position: relative;
+`
+const ContactCardContainer = styled.div`
+  display: inline-flex;
+  min-width: 100%;
+  position: absolute;
+  top: ${props => (props.index % 2 === 0 ? "0" : "180px")};
+`
+
+const MemberImg = styled.div`
```

Comment:
I see now that these might just be copy-pasted, could you deduplicate them please? Just pull out all the shared ones into their own component file

Reply:
Yeah I added a file to move the duplicated ones to. I haven't really touched the advisor parts yet so I'm not sure what will be duplicated just yet, but gonna move them to the team-shared-styles.js file as I see them.

Reply:
If they're not being used, would you mind just deleting them then? It just reduces the amount of cleanup for later on Github and you can always copy+paste again